.github/workflows/build.yml

```diff
@@ -205,6 +205,61 @@ jobs:
         run: |
           cat ./build/meson-logs/testlog.txt
 
+  linux-ffmpeg-4-4:
+    runs-on: ubuntu-22.04
+    container:
+      image: alpine:3.15
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install dependencies
+        run: |
+          apk update
+          apk add \
+              binutils \
+              ffmpeg-dev \
+              gcc \
+              git \
+              g++ \
+              glslang-dev \
+              libass-dev \
+              lcms2-dev \
+              lua5.2-dev \
+              pipewire-dev \
+              pulseaudio-dev \
+              libva-dev \
+              libx11-dev \
+              libxkbcommon-dev \
+              libxrandr-dev \
+              libxpresent-dev \
+              libxscrnsaver-dev \
+              musl-dev \
+              py3-pip \
+              samurai \
```

Comment:
i don't think meson is actually using this

Reply:
It is for sure

Reply:
?
![grafik](https://github.com/mpv-player/mpv/assets/1042418/8d3aca53-1efd-448f-b4cb-7ec24f43e3ec)

Reply:
samurai provides that, apparently. At least when I removed it meson couldn't find ninja